akita/src/api/entity-store.ts

```diff
@@ -71,12 +71,13 @@ export class EntityStore<S extends EntityState<E>, E> extends Store<S> {
    * @example
    * this.store.add([Entity, Entity]);
    * this.store.add(Entity);
+   * this.store.add(Entity, { prepend: true });
    */
-  add(entities: E[] | E) {
+  add(entities: E[] | E, params ?: AddParams) {
```

Comment:
Now it's great. One final thing, can you change `params` to `options` and `AddParams` to `AddOptions`? thanks.